content/platform/data-services-cloud-console-api/home.md

```diff
@@ -0,0 +1,176 @@
+
+### Supported API Categories (Services)
+
+The API categories for DSCC will grow in accordance to the expansion of the DSCC services that are going to released in future. Current services that are available, as recorded today:
+
+#### Common (Alletra-6K, Alletra-9K, Primera, Nimble)
+
+1. Authentication
+2. Tasks
+3. Event Audit
+4. AuthZ (User RBAC permissions)
+5. Issues
+6. controllers
+7. host-initiator-groups
+8. Host-initiators
+9. ports
+10. shelves
+11. storage-pools
+12. storage-systems
+13. system-settings
+14. volume-sets
+15. volumes
+
+#### Alletra-6K or Nimble
+
+1. protection-templates
+2. disks
+
+### Versioning
+
+The major version number will be provided in the resource path as "v1" in this example:
+
+```md
+/api/v1/<resource group>/...
+```
+
+Some examples of these resource paths that contain several resource groups under the same root:
+
+```markdown
+/api/v1/storage-systems/...
+
+/api/v1/controllers/...
+
+/api/v1/volumes/...
+```
+
+Existing clients will be able to maintain the backward compatibility from the higher major version incremental, and adopt newly introduced API.  However, both the new and old version of API will be supported until the announcement of the deprecation for the old version of API. Nonetheless, the older major version will always be frozen with exception of bug fixes. There will also be an announcement of deprecation in the header and sunset header. 
+
+```md
+/api/v1/<resource group>/...
+
+/api/v2/<resource group>/...
+```
+
+### HTTP Request Methods
+
+| HTTP Verbs | Description                                                         |
+| ---------- | ------------------------------------------------------------------- |
+| GET        | Retrieves target resource                                           |
+| POST       | Creates an entity or changes state                                  |
+| PUT        | Replaces target resource with data part of the HTTP Request payload |
+| DELETE     | Remove the target source                                            |
+
+### Authorization through OAuth2 Client-Credential work-flow
+
+The client's application can issue REST API request using the access-token as the bearer of the token.  The client obtains this access-token from the after the client successfully authenticate through an associated customer's application credential (client-id and client-secret). This application credential is created by the DSCC user who has the permission to access resources (such as controllers, volumes etc.) under the DSCC instances. This access-token expiration time, by default, is set for 7200 seconds (2 hours). When the the resource server see this expired access token, and return 0x401 response (not authorized).  The client must then reauthenticate using the associated client-id and client-secret to obtain the next access-token to use for the next REST API request.
```

Comment:
add the word "a" prior to "REST API request" in the first sentence.
In the second sentence, there appears to be something missing where it says " obtains this access-token **from the after** the". It obtains it from the WHAT? Also, in general, I don't believe there should be a hyphen between "access" and "token" and you don't always use it. I would remove the hyphen.

The second to the last sentence has some issues. I think it should probably read:
When the resource servers see**s** this expired access token, **it returns a** 0x401 response (not authorized).